app/models/user.rb

```diff
@@ -4,6 +4,8 @@ class User < ApplicationRecord
 
   belongs_to :credentials, :polymorphic => true
 
+  enum language: [:nl, :en]
```

Comment:
Why is the enum present in both user and member?

Reply:
This was a leftover from previous code, was removed